### PR TITLE
Add two articles with Rstudio/Github/Git tutorials

### PR DESCRIPTION
**Pull request by novica**

### CODE REVIEW

**Comment by novica** [comment]
I updated the text following your comments.

@hkaspersen woudl you be willing to check out the git part?

File: docs/source/tutorials/rstudio_github.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+The token will be shown in the browser. Keep that open for the time being.
+
+In RStudio install `devtools` and `usethis` packages: `install.packages(“devtools”)`; `usethis: install.packages(“usethis”)`.
+
+In the R terminal in RStudio type and run `usethis::edit_r_environ()` – this will open a blank `.Renviron` file (see [here](https://support.posit.co/hc/en-us/articles/360047157094-Managing-R-with-Rprofile-Renviron-Rprofile-site-Renviron-site-rsession-conf-and-repos-conf) if you want to learn more about it).
+
+In the empty file write GITHUB_PAT="" and paste the token from the browser between the quotation marks. Save the `.Renviron` file. Close RStudio or restart the R session. Close the browser tab that has the token.
+
+All done!
```
Comment (hkaspersen) [inline docs/source/tutorials/rstudio_github.md:45]: Perhaps add a section describing how to use git outside Rstudio? I usually don't use the Rstudio IDE for git stuff, I only use Git Bash. Not perhaps strictly necessary for this tutorial, but might be useful as an alternative!

**Review by hkaspersen — CHANGES_REQUESTED** [review]
Would be nice with a small section for git outside Rstudio

**Comment by novica** [comment]
> This looks great! Would be nice with a small section for git outside Rstudio as an alternative though, but otherwise this is good.

I thinik a separate article for git will make much more sense than having in in a Rstudio related one.  I can do that in the coming days.

**Review by hkaspersen — APPROVED** [review]

**Pull request merged**